pandas/tests/frame/methods/test_map.py

```diff
@@ -0,0 +1,24 @@
+import pandas as pd
+import pandas._testing as tm
+
+
+class TestMap:
```

Comment:
we have extensive map tests here: pandas/tests/apply/test_series_apply.py

see if you can find something similar. you can use the fixture `any_string_dtype` to parmeterize this over various kinds of strings.

Reply:
Ok moved. Should the fixture be used like this? 
```
# parameter
test_series_map_stringdtype(any_string_dtype):
    ser1 = Series(
        # ...
        # then set here for all Series
        dtype=any_string_dtype,
    )
 ```